src/ui_ng/lib/src/helm-chart/helm-chart.component.ts

```diff
@@ -106,10 +117,22 @@ export class HelmChartComponent implements OnInit {
     this.chartClickEvt.emit(item.name);
   }
 
+  resetUploadForm() {
+    this.chartFile = null;
+    this.provFile = null;
+    this.uploadForm.reset();
+  }
+
   onChartUpload() {
+    this.resetUploadForm();
     this.isUploadModalOpen = true;
   }
 
+  cancelUpload() {
+    this.resetUploadForm();
```

Comment:
Do we really need to reset uploading forms two times?

Reply:
No, because only once can assure the whole system works fine.
Yes, because we don't know how the user operation our system and we only can do is  to assure that every operation will work.